window/src/os/windows/window.rs

```diff
@@ -390,6 +396,15 @@ fn decorations_to_style(decorations: WindowDecorations) -> u32 {
     }
 }
 
+unsafe fn get_primary_monitor_dpi() -> u32 {
+    let zero_point = POINT { x: 0, y: 0 };
+    let primary = MonitorFromPoint(zero_point, MONITOR_DEFAULTTOPRIMARY);
```

Comment:
I think switching this to `MonitorFromWindow(std::ptr::null_mut(), MONITOR_DEFAULTTOPRIMARY)` would be more consistent with the logic in `create_window` and remove issues with `0,0` not being on the primary monitor in some unusual user-configurations